go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -284,7 +290,11 @@ func CreateLoadBalancerVIPs(lb string,
 		}
 		vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 		lbTarget := fmt.Sprintf(`vips:"%s"="%s"`, vip, strings.Join(targets, ","))
-		txn.Add("set", "load_balancer", lb, lbTarget)
+		request := []string{"set", "load_balancer", lb, lbTarget}
```

Comment:
@trozet why the `request` variable? Was this just for line-length? You could just do `AddOrCommit(args ...string)`.

Reply:
it was just to provide better readability